src/routes/game/components/dialogs/components/ScrapDialog.vue

```diff
@@ -4,10 +4,56 @@
     v-model="show"
     :scrollable="true"
     :persistent="false"
+    :attach="false"
   >
     <template #activator="{ props }">
-      <span v-bind="props">
-        <slot name="activator" />
```

Comment:
No longer need an activator slot because there's only one consumer of this component and ScrapDialg controls its own activator